src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -844,12 +845,14 @@ impl<'a, 'tcx> Visitor<'tcx> for Qualifier<'a, 'tcx, 'tcx> {
 
             // Deny *any* live drops anywhere other than functions.
             if self.mode != Mode::Fn {
-                // HACK(eddyb) Emulate a bit of dataflow analysis,
-                // conservatively, that drop elaboration will do.
```

Comment:
Need to keep this comment around because this is still emulated until we use something more like what drop elaboration does.